### DISCUSSION TITLE
ProTable - 高级表格 我想设置初始化时不显示在table，但是又想不勾选显示在列设置中如何做？

### DISCUSSION THREAD

**Discussion by AlbertXiaoPeng** [ant-design/pro-components#8215] [Q&A]
期望初始化时：
![image](https://github.com/ant-design/pro-components/assets/44151574/1ae909eb-25c0-4c2c-bdf8-f9ab9689910c)

通过这三个参数设置并没有成功，要不都不显示（table没有，列设置也没有），要不都显示并勾选（table显示，列设置也显示并勾选），没法设置成我期望的样子

![image](https://github.com/ant-design/pro-components/assets/44151574/e3deaae7-faf3-43f9-8028-0905552c70a5)

**Comment by esclipse** [comment]
列设置是可以受控展示的，默认是都勾选。参考这个文档 https://procomponents.ant.design/components/table?tab=api&current=1&pageSize=5#%E5%8F%97%E6%8E%A7%E7%9A%84%E8%A1%A8%E6%A0%BC%E8%AE%BE%E7%BD%AE%E6%A0%8F
